.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Related Issue
+
+- Closes: #[issue number] (Link the issue this pull request addresses)
```

Comment:
I think we have to get rid of the space between the brackets here `] (` for it to work. Also I think github automatically makes a formatted link if you just link the issue so that could work as well

Reply:
Thanks for catching that, @kadenzipfel ! You're absolutely right on both counts.

Here's the updated template with the space removed and using GitHub's automatic linking:

```suggestion
- Closes: #[issue number]
```

This makes the template cleaner and avoids redundancy. please let me know if it's fine or not

Reply:
I actually think it would be better to have it like:
```
- Closes: (Link the issue this pull request addresses)
```
so the user can just include the link there and be done with it